documentation/AZURE.md

```diff
@@ -0,0 +1,66 @@
+Azure credential and setup
+==========================
+
+Setup
+-----
+* Create an application ( [visual guide](https://docs.microsoft.com/en-in/azure/azure-resource-manager/resource-group-create-service-principal-portal#create-an-active-directory-application) )<br />
+	Search for **App registrations** in the top search bar and select the first result.<br />
+	Select **New application registration**.<br />
+	Fill in the application form:
+
+		Name: StreisandAuth
+		Application type: Web app / API
+		Sign-on URL: http://StreisandAuth
+
+* Ensure that the application can create Azure instances ( [visual guide](https://docs.microsoft.com/en-in/azure/azure-resource-manager/resource-group-create-service-principal-portal#assign-application-to-role) )<br />
+	Search for **Subscriptions** in the top search bar and select the first result.<br />
+	Choose the desired subscription.<br />
+	Select **Access control (IAM)** from the newly opened menu.<br />
+	Add:
+
+		Role: Contributor
+		Select: StreisandAuth
+
+Credentials
+-----------
+File format ( [source](https://docs.ansible.com/ansible/guide_azure.html#storing-in-a-file) ):

```

Comment:
It might be worth adding a quick note here that the `xxxxx` patterns will have to be replaced. I know this seems obvious but experience has shown that the general public has a way of surprising you about what is/isn't obvious :-)